packages/driver/cypress/support/defaults.js

```diff
@@ -11,6 +11,12 @@ if (!isActuallyInteractive) {
   Cypress.config('retries', 2)
 }
 
+// set an example serializable and unserializable value to validate multi-domain config/env syncing on initial spec bridge creation
+Cypress.config('foo', 'bar')
```

Comment:
This smells bad to me, but I am not sure of any other ways to deterministically test the `config` and `env` syncing for multi-domain. Any ideas?

Reply:
You could remove the `iframe` in the test so it would get recreated. Then you could set the custom values in the test instead of here.

Reply:
I don't really understand. What's non-deterministic about setting these in the spec file itself?

Reply:
@mschile looks like something like `window.top.document.querySelector('[id="Cypress (foobar.com)"]').remove()` does the trick! Thank you. I'll make the selector less brittle but I like this much more than polluting the generic defaults.

Reply:
I just merged my PR to change the id of the iframe from `Cypress (foobar.com)` to `Spec Bridge: (foobar.com)` so you'll probably need to update the selector for that change.

Reply:
wound up not needing it with the bi-directional sync changes